pkgs/build-support/php/build-composer-project.nix

```diff
@@ -73,6 +73,11 @@ let
         composerStrictValidation = previousAttrs.composerStrictValidation or true;
       };
 
+      passthru = (previousAttrs.passthru or {}) // {
+        php = phpDrv;
+        vendor = finalAttrs.composerRepository;
```

Comment:
Why not just use `composerRepository` attribute directly?

Reply:
Why not actually. I liked the idea of referencing the vendor using `.vendor`.

Reply:
I'll follow your advice and remove it then.

Reply:
Generally, we try to have a single source of truth – it can be confusing seeing the same attribute twice (e.g. in repl) and overriding `vendor` will make it out of sync, the `composerRepository` not applying.

Reply:
Okidocky, will do the change, thank you for correcting me.